<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TMainOperatorsSettingsWidget</class>
 <widget class="QWidget" name="TMainOperatorsSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>255</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QPushButton" name="centBtn">
     <property name="minimumSize">
      <size>
       <width>65</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>%</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="addBtn">
     <property name="minimumSize">
      <size>
       <width>65</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>+</string>
     </property>
     <property name="shortcut">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="rootBtn">
     <property name="minimumSize">
      <size>
       <width>65</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>Root</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="cBtn">
     <property name="minimumSize">
      <size>
       <width>65</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>c</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="multiplyBtn">
     <property name="minimumSize">
      <size>
       <width>65</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>*</string>
     </property>
     <property name="shortcut">
      <string>*</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="subtractBtn">
     <property name="minimumSize">
      <size>
       <width>65</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>-</string>
     </property>
     <property name="shortcut">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="ceBtn">
     <property name="minimumSize">
      <size>
       <width>65</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>ce</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="clearAllBtn">
     <property name="minimumSize">
      <size>
       <width>65</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>Clear all</string>
     </property>
     <property name="shortcut">
      <string>Backspace</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="inverseBtn">
     <property name="minimumSize">
      <size>
       <width>65</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>Inverse</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="divisionBtn">
     <property name="minimumSize">
      <size>
       <width>65</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>/</string>
     </property>
     <property name="shortcut">
      <string>/</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="squareOfNumberBtn">
     <property name="minimumSize">
      <size>
       <width>65</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>X2</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="equalBtn">
     <property name="minimumSize">
      <size>
       <width>65</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>=</string>
     </property>
     <property name="shortcut">
      <string>Enter</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
